### DISCUSSION TITLE
DaxStudio compatible with screen readers or other accessibility software

### DISCUSSION THREAD

**Discussion by nhsbsa-wicoo** [DaxStudio/DaxStudio#1263] [Q&A]
Hi, I've had a look through the docs and the blog but can't seem to find anything related to accessibility software. Has anyone used it with a screen reader or speech recognition software? How compatible is it?

**Comment by dgosbell** [comment]
I don't really know it's not something I've tested for. If you do try it I would be interested to hear how it goes.
